src/services/services.ts

```diff
@@ -2437,7 +2437,7 @@ export function createLanguageService(
     }
 
     function getDocCommentTemplateAtPosition(fileName: string, position: number, options?: DocCommentTemplateOptions): TextInsertion | undefined {
-        return JsDoc.getDocCommentTemplateAtPosition(getNewLineOrDefaultFromHost(host), syntaxTreeCache.getCurrentSourceFile(fileName), position, options);
+        return JsDoc.getDocCommentTemplateAtPosition(getNewLineOrDefaultFromHost(host, /*formatSettings*/ undefined), syntaxTreeCache.getCurrentSourceFile(fileName), position, options);

```

Comment:
Not that you should fix it in this PR, but it seems like a bug that we don't have this for JSDoc comments.